runtime/verutil/chverify.c

```diff
@@ -150,33 +150,38 @@ isInitOrClinitImpl (J9CfrConstantPoolInfo * info)
 		if (J9UTF8_DATA_EQUALS("<clinit>", 8, name, info->slot1)) {
 			return CFR_METHOD_NAME_CLINIT;
 		}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) && defined(J9VM_OPT_NEW_FACTORY_METHOD)
+		if (J9UTF8_DATA_EQUALS("<new>", 5, name, info->slot1)) {
+			return CFR_METHOD_NAME_NEW;
+		}
+#endif /* #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) && defined(J9VM_OPT_NEW_FACTORY_METHOD) */
 		return CFR_METHOD_NAME_INVALID;
 	}
 	return 0; /* not <init> or <clinit> */
 }
 
 /**
- * Determine if this name is either "<init>" or "<clinit>".
+ * Determine if this name is "<init>", "<clinit>", or "<new>".
  *
- * @returns 0 if name is a normal name, 1 if '<init>' and 2 if '<clinit>' , -1 if it starts with '<' but is not a valid class name.
- * @note result is positive if the name is "<init>" or "<clinit>", result is negative if the name is illegal
+ * @returns 0 if name is a normal name, 1 if '<init>', 2 if '<clinit>', 3 if '<new>', and -1 if it starts with '<' but is not a valid class name.
```

Comment:
Please use `CFR_METHOD_NAME_INIT`, `CFR_METHOD_NAME_CLINIT`, `CFR_METHOD_NAME_NEW` and `CFR_METHOD_NAME_INVALID`, instead of the actual values in this comment. The same applies to comment of `bcvCheckMethodName()` below.